middleware/proxy/upstream.go

```diff
@@ -175,6 +181,12 @@ func parseBlock(c *parse.Dispenser, u *staticUpstream) error {
 			return c.ArgErr()
 		}
 		u.IgnoredSubPaths = ignoredPaths
+	case "proxy_redirect":
```

Comment:
i think #444 is same  problem with this.

example(a svn server run in docker):

```javascript
        proxy /svn http://localhost:8881/svn {
                without /svn
                proxy_header Host {host}
                proxy_redirect http://mydomain https://mydomain
        }
```

without ```proxy_header Host {host}```, ```https://mydomain/svn/myproject``` will redirect to ```http://localhost:8881/svn/myproject``` 

without ```proxy_redirect http://mydomain https://mydomain```, ```https://mydomain/svn/myproject``` will redirect to ```http://mydomain/svn/myproject```.

PS: very sorry for my poor english =.=!!!